### PR TITLE
community picks new page

### PR DESCRIPTION
**Pull request by GrahamTheDev**

### CODE REVIEW

File: docs/key-features/community-picks.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+
+
+
+### Community picks feed
+All of the articles submitted by the community are available in the [Community Picks feed.](https://app.daily.dev/sources/community)
+
+![](https://daily-now-res.cloudinary.com/image/upload/v1657719387/cp-3-community-feed_enjtux.png)
+
+## Submitting a Community Pick
```
Comment (idoshamun) [inline docs/key-features/community-picks.md:35]: It think it's more aligned with our terminology

```suggestion
## Scouting a Community Pick
```

File: docs/key-features/community-picks.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+To be able to submit a Community Pick you need to have at least 250 reputation on daily.dev
+
+[Learn how to gain reputation](https://docs.daily.dev/docs/how-does-daily-dev-work/reputation) and how reputation works.
+
+If you do not have enough reputation to submit a link you will see a message telling you that you do not have enough reputation and the submit button will be disabled.
+
+![](https://daily-now-res.cloudinary.com/image/upload/v1657719512/cp-4-not-enough-permissions_zadogk.jpg)
+
+### How to submit a link
+On the menu on the left you will see a new section "Contribute" containing a button "Submit article".
```
Comment (idoshamun) [inline docs/key-features/community-picks.md:48]: `new` is a relative term that doesn't hold in the test of time. I recommend removing it

File: docs/key-features/community-picks.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+Submitting a link to the daily.dev feed via Community Picks is straight forward, however you may not have access to Community Picks yet if you do not have enough reputation.
+
+### Reputation Requirements
+To be able to submit a Community Pick you need to have at least 250 reputation on daily.dev
+
+[Learn how to gain reputation](https://docs.daily.dev/docs/how-does-daily-dev-work/reputation) and how reputation works.
+
+If you do not have enough reputation to submit a link you will see a message telling you that you do not have enough reputation and the submit button will be disabled.
+
+![](https://daily-now-res.cloudinary.com/image/upload/v1657719512/cp-4-not-enough-permissions_zadogk.jpg)
```
Comment (idoshamun) [inline docs/key-features/community-picks.md:45]: The image looks buggy because it says 4/3

File: docs/key-features/community-picks.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| unfortunately there was an error and we were unable to gather the required information from the URL submitted to add it to our feed. | If a site has an unusual structure our crawler may not be able to parse the page correctly to create the tags and TLDR. |
+
+## Gaining Reputation
+Not only do you help the community by sharing useful and interesting articles, you can gain more reputation too!
+
+You gain reputation on any link you submit when it receives an upvote, as well as the post author if they have a profile on daily.dev.
+
+We plan on releasing more features that are dependant on reputation in the future, so this is a great way to gain reputation while helping the community.
+
+## What do we want to see?
```
Comment (idoshamun) [inline docs/key-features/community-picks.md:99]: I'm not sure people will read this far and I want to make sure they know our guidelines before they understand how to submit.

An important point that is missing: "don't submit your own articles".

File: docs/key-features/community-picks.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| you can’t submit your own articles as community picks, please suggest articles by other people. | Community Picks are not designed for self promotion, instead try submitting articles written by others that you think will benefit the community. |
+| unfortunately the article you submitted is behind a paywall, so we cannot add it to the daily.dev feed. | We try our best to make sure all articles shared are free for the community. If we detect a paywall then an article will not be accepted for the feed. |
+| unfortunately there was an error and we were unable to gather the required information from the URL submitted to add it to our feed. | If a site has an unusual structure our crawler may not be able to parse the page correctly to create the tags and TLDR. |
+
+## Gaining Reputation
+Not only do you help the community by sharing useful and interesting articles, you can gain more reputation too!
+
+You gain reputation on any link you submit when it receives an upvote, as well as the post author if they have a profile on daily.dev.
```
Comment (idoshamun) [inline docs/key-features/community-picks.md:95]: There's a nuance to this. The upvoter needs to have a certain amount of reputation as well.
It reminds me that we need to update the reputation page as well

**Review by idoshamun — CHANGES_REQUESTED** [review]
Overall it looks fantastic! I found some missing pieces that I'd love to have

File: docs/key-features/community-picks.md (hunk 1 above)
Comment (GrahamTheDev) [reply docs/key-features/community-picks.md:32]: Suggest ## Submitting / Scouting a Community Pick as "Scouting" without explanation might not be clear as an action.

File: docs/key-features/community-picks.md (hunk 2 above)
Comment (GrahamTheDev) [reply docs/key-features/community-picks.md:45]: Agreed!

File: docs/key-features/community-picks.md (hunk 3 above)
Comment (GrahamTheDev) [reply docs/key-features/community-picks.md:42]: Ah I will fix that, I even noticed it when Chris shared it and then didn't fix, I will fix this after launch

File: docs/key-features/community-picks.md (hunk 4 above)
Comment (GrahamTheDev) [reply docs/key-features/community-picks.md:105]: I have added a section to the info part to cover this

**Pull request merged**